charts/rbacdefinitions/templates/rbacdefinitions.yaml

```diff
@@ -0,0 +1,13 @@
+{{- range .Values.rbacDefinitions }}
+---
+# raw rbacDefinition: {{ . | toJson }}
```

Comment:
```suggestion
```

(i forgot to remove this debug line in cert-manager-issuers but it's wrong there as  well)

Reply:
haha I was unsure why it was there, so I just left it. gonna remove it.